src/components/MaterialRulesDialog.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import { Button, Dialog, DialogActions, DialogContent, DialogContentText,
+    DialogTitle, withMobileDialog, Typography
+} from '@material-ui/core';
+
+const styles = theme => ({
+  bold: {
+    fontWeight: theme.typography.fontWeightMedium,
+  },
+});
+
+class KotbarRulesDialog extends React.Component {
+
+  render() {
+    const { classes, fullScreen, open, onAccept, onClose } = this.props;
+
+    return (
+      <Dialog
+        fullWidth
+        fullScreen={fullScreen}
+        open={open}
+        onClose={onClose}
+        aria-labelledby="kotbarrules-dialog-title"
+        aria-describedby="kotbarrules-dialog-description"
+      >
+        <DialogTitle id="kotbarrules-dialog-title">Materiaal Rules</DialogTitle>
+        <DialogContent>
+          <DialogContentText variant="body2">Materiaalkot:</DialogContentText>
+          <ul>
+            <li>
+              <Typography>
+                Geef de <span className={classes.bold}>sleutel</span> af aan de materiaalverantwoordelijke (= niet onderling doorgeven).
+              </Typography>
+            </li>
+            <li>
+              <Typography>
+                Zet de spullen terug op de <span className={classes.bold}>voorziene plaats</span>.
+              </Typography>
+            </li>
+            <li>
+              <Typography>
+                Meld aan de verantwoordelijke indien iets <span className={classes.bold}>kapot of aan vervanging toe</span> is.
+              </Typography>
+            </li>
+            <li>
+              <Typography>
+                Alles wat je in het materiaalkot plaatst is <span className={classes.bold}>gemeenschappelijk</span>. Zet dus geen persoonlijke spullen hier.
+              </Typography>
+            </li>
+          </ul>
+          <DialogContentText variant="body2">Beamer:</DialogContentText>
```

Comment:
Ik denk dat het vooral ook voor de beamer belangrijk is dat die steeds langs de materiaalverantwoordelijke passeert